appserver/extras/docker-images/server-full/src/main/docker/bin/startInForeground.sh

```diff
@@ -41,8 +43,9 @@ if [ -z $DOMAIN_NAME ]; then echo "Variable DOMAIN_NAME is not set."; exit 1; fi
 # Create pre and post boot command files if they don't exist
 touch $POSTBOOT_COMMANDS
```

Comment:
This isn't necessary since POSTOOT_COMMANDS file is now optional and isn't used to start the server.
```suggestion
```